packages/plugins/components/src/NavigationDropdown/NavigationDropdown.test.tsx

```diff
@@ -0,0 +1,46 @@
+import { ThemeProvider } from '@material-ui/styles';
+import * as React from 'react';
+import { render, within, fireEvent, waitFor } from '@testing-library/react';
+
+import { createMuiTheme } from '@material-ui/core';
+import { NavigationDropdown } from '.';
+
+const menuItems = [
+  {
+    title: 'Projects',
+    url: '/console',
+  },
+  {
+    title: 'Org',
+    url: '/dashboard',
+  },
+];
+const muiTheme = createMuiTheme({});
+
+describe('NavigationDopdown', () => {
+  it('dropdown opens on click', async () => {
+    const { getByRole } = render(
+      <ThemeProvider theme={muiTheme}>
+        <NavigationDropdown
+          items={menuItems}
+          config={{ headerFontFamily: `"Open Sans", helvetica, arial, sans-serif` }}
+        />
+      </ThemeProvider>,
+    );
+
+    const defaultButton = getByRole('button');
+    expect(defaultButton).not.toBeNull();
```

Comment:
`getByRole` doesn't return `null` if it can't find an element, but throws an error instead - it's safer to use `queryByRole` for these types of checks